addon/helpers/d3-area.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+import { area } from 'd3-shape';
+
+export function d3Area( [ xScale, yScale ], { xAccessor, yAccessor } ) {
+  let yMin = yScale.domain()[0];
+  let xFn  = (d) => xScale(xAccessor(d));
```

Comment:
No reason other than time. I'm planning on making a more complex chart in the near future and I will probably work those in then.